src/v/redpanda/admin/api-doc/transform.json

```diff
@@ -0,0 +1,155 @@
+{
+  "apiVersion": "0.0.1",
+  "swaggerVersion": "1.2",
+  "basePath": "/v1",
+  "resourcePath": "/transform",
+  "apis": [
+    {
+      "path": "/v1/transform/deploy",
+      "operations": [
+        {
+          "method": "POST",
+          "summary": "Deploy a transform based on the provided metadata and wasm binary in the request payload. The payload should be a json `deploy_transform_request` object immediately followed by the wasm binary without any delimiters.",
+          "nickname": "deploy_transform",
+          "consumes": [
+            "application/json+wasm"
+          ],
+          "type": "void",
+          "produces": [
+            "application/json"
+          ]
+        }
+      ]
+    },
+    {
+      "path": "/v1/transform/delete",
+      "operations": [
+        {
+          "method": "POST",
+          "summary": "Delete a transform based on the provided name in the request payload.",
+          "type": "void",
+          "nickname": "delete_transform",
+          "consumes": [
+            "application/json"
+          ],
+          "produces": [
+            "application/json"
+          ]
+        }
+      ]
+    },
+    {
+      "path": "/v1/transform/list",
+      "operations": [
+        {
+          "method": "GET",
+          "summary": "List all transforms, the status report contains a cluster wide aggregated view of the live transform execution state.",
+          "nickname": "list_transforms",
+          "produces": [
+            "application/json"
+          ]
+        }
+      ]
```

Comment:
What is the return type here?

Reply:
https://docs.google.com/document/d/13w8yyMaeFZcCNB6EqgIzDwRFiAGqHY5Aler0SPyFmT4/edit#heading=h.b0l6hh2ln6o0

empty response body on success. Is there a way to specify that in this file?

Reply:
You could also specifically type the response. Take a look at the response definitions for `get_cluster_config` within cluster_config.json

Reply:
Thanks I typed the response for list (it's an array of transform metadata)